utilities_common/platform_sfputil_helper.py

```diff
@@ -133,7 +133,7 @@ def is_rj45_port(port_name):
 
```

Comment:
Did a bit of debugging and see that the below code section 
```
        if not platform_porttab_mapping_read:
            platform_sfputil_read_porttab_mappings()
```
is actually creating this logs which @mlok-nokia was pointing out
```
Aug 25 05:50:23.693785 str2--lc1-1 ERR Chassis: :- initializeGlobalConfig: SonicDBConfig Global config is already initialized
```

Reply:
thanks. I'll check it.

Reply:
@judyjoseph if possible, can you share more details, like call stack? I don't have a multi-chassis testbed, it will be time-consuming for me to find it out. thank you.